package.json

```diff
@@ -8,7 +8,9 @@
     "watch": "webpack --watch",
     "heroku-postbuild": "webpack",
     "test": "mocha-webpack \"./frontend/test/**/*.test.js\" --webpack-config webpack-test.config.js --require ./frontend/test/setup.js",
-    "test-watch": "mocha-webpack --watch \"./frontend/test/**/*.test.js\" --webpack-config webpack-test.config.js --require ./frontend/test/setup.js"
+    "test-watch": "mocha-webpack --watch \"./frontend/test/**/*.test.js\" --webpack-config webpack-test.config.js --require ./frontend/test/setup.js",
+    "createDB": "sequelize db:create",
```

Comment:
let's keep these with dashes to stay consistent, the other commands are dashed instead of camel cased